src/views/v2/rsvp/form/fields/email.php

```diff
@@ -34,8 +34,7 @@ class="tribe-common-b2--min-medium tribe-tickets__form-field-label"
 	</label>
 	<input
 		type="email"
-		id="tribe-tickets-rsvp-email"
```

Comment:
doesn't the label above need this (for accessibility and to link the label to the input, such as clicking on the label)?